lib/my_scaffold.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:flutter/material.dart';
+
+class MyScaffold extends StatelessWidget {
```

Comment:
Ich bin gegen diese Klasse, wenn sie nicht für später irgendeinen besonderen Sinn hat. Soweit ich das sehe wollt ihr damit die eine Zeile appBar: AppBar(...) einsparen?? Als Tradeoff dafür fügt es eine unnötige Komplexitätsebene hinzu und macht den Code für alle schwerer zu lesen (Wenn ich eine ganz normale Page sehe würde ich mich da erstmal fragen ob wir uns was komplett eigenes gebaut haben und dann müsste ich in diese Klasse reinschauen, anstatt direkt zu sehen das ist ein ganz normales Scaffold), und am Ende ist der Code weniger maintainable weil vielleicht doch mal jemand (AG?) eine der anderen 37.000 Funktionalitäten des Scaffolds benutzen will und dann wird in der Hälfte der App Scaffold und in der anderen MyScaffold benutzt oder die MyScaffold-Klasse wird aufgebohrt bis sie quasi eine exakte Kopie ist

Reply:
ich will halt nicht jedes mal eine "neue" AppBar da hinpacken, mit immer dem gleichen Text... I guess wir können eine Klasse MotisAppBar machen und die in jedes Scaffold einfügen

Reply:
Ich würde nicht sagen dass am Ende in jeder AppBar dasselbe steht, steht da nicht immer der Titel der Seite würdest du sagen?

Reply:
ja das wäre halt die Frage... wenn wir das so machen, dann brauchen wir das natürlich absolut nicht

Reply:
Ich kenne keine App die das so macht und ich denke es ist auf jeden Fall sinnvoll oben anzuzeigen auf welcher Seite man ist, vor allem wegen unserer etwas komplizierten Mitfahrer/Fahrer-Situation. Für Suchergebnisse etc. fände ich das auch etwas weird, aber da wäre ich komfortabler damit wenn in der AppBar einfach gar nichts steht, also AppBar(), da braucht man auch keine neue Klasse

Reply:
Ist eigentlich eine Gute idee. Wäre dann auch einfacher die Seiten create und search farblich oder so von einander zu trennen.